<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindowClass</class>
 <widget class="QMainWindow" name="mainwindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>637</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>mainwindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>581</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>271</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>数字与文字识别系统</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>331</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string>图片预览</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>460</y>
        <width>91</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>修改图片</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>460</y>
        <width>271</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>上传文件</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>200</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>识别按钮</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>310</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>chatbox</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>440</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>退出系统</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_history">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>70</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>历史记录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_toPage4">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>图像处理系统</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>351</width>
        <height>241</height>
       </rect>
      </property>
      <property name="text">
       <string>图片预览</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>280</y>
        <width>291</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>识别文字的展示和输出</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>390</y>
        <width>471</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>历史记录</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prompt">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>531</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_history">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ollama">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>230</y>
        <width>101</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_result">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>351</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_favorites">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>221</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_addFavorite">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_removeFavorite">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>180</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_search">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>350</y>
        <width>421</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_search">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>430</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>联网搜索</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rag">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>460</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>RAG流程</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_results">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>221</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>170</y>
        <width>161</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_gaussian">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>60</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>亮度归一化</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_gray">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>130</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>sodel 边缘检测</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_canny">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>200</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>K-Means 聚类分析</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_effectA">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_effectB">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>340</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_effectC">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>400</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_effectD">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_effectE">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>520</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>784</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
